<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scanTab</class>
 <widget class="QWidget" name="scanTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="mainFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="virusFoundLabel">
          <property name="text">
           <string>When a virus is found</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="virusFoundComboBox">
          <item>
           <property name="text">
            <string>just report</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/icons/includeexclude.png</normaloff>:/icons/icons/includeexclude.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>remove file (be careful)</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/icons/trash-can.png</normaloff>:/icons/icons/trash-can.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>move file to qurantine folder</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/icons/edit-cut.png</normaloff>:/icons/icons/edit-cut.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>copy file to quarantine folder</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/icons/edit-copy.png</normaloff>:/icons/icons/edit-copy.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="recursivCheckBox">
          <property name="text">
           <string>scan recursive</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QTreeView" name="treeView">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="startScanButton">
        <property name="text">
         <string>start</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/icons/start.png</normaloff>:/icons/icons/start.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="stopScanButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>stop</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>168</width>
           <height>277</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="deviceFrame">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="pathFrame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="2" column="0">
                 <widget class="QPushButton" name="homePathButton">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">text-align:left</string>
                  </property>
                  <property name="text">
                   <string>Home</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../resources.qrc">
                    <normaloff>:/icons/icons/homefolder.png</normaloff>:/icons/icons/homefolder.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>28</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="basePathButton">
                  <property name="styleSheet">
                   <string notr="true">text-align:left</string>
                  </property>
                  <property name="text">
                   <string>Root</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../resources.qrc">
                    <normaloff>:/icons/icons/basefolder.png</normaloff>:/icons/icons/basefolder.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>28</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="pathLabel">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background:#c0c0c0;padding:4px;border-radius:5px;</string>
                  </property>
                  <property name="text">
                   <string>Pathes</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="devicesFrame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="deviceLabel">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background:#c0c0c0;padding:4px;border-radius:5px;</string>
                  </property>
                  <property name="text">
                   <string>Devices</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QFrame" name="logStatusFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-weight:bold</string>
           </property>
           <property name="text">
            <string>Signature Info</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="logHeaderLabel">
           <property name="styleSheet">
            <string notr="true">font-weight:bold</string>
           </property>
           <property name="text">
            <string>Status / Log-Messages</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTextBrowser" name="versionTextBrowser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPlainTextEdit" name="logPlainTextEdit">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="centerOnScroll">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="currentFileLabel">
           <property name="styleSheet">
            <string notr="true">padding:5px</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Info: </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startScanButton</sender>
   <signal>clicked()</signal>
   <receiver>scanTab</receiver>
   <slot>slot_scanButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>834</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopScanButton</sender>
   <signal>clicked()</signal>
   <receiver>scanTab</receiver>
   <slot>slot_abortScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>920</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recursivCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>scanTab</receiver>
   <slot>slot_recursivScanCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>virusFoundComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>scanTab</receiver>
   <slot>slot_virusFoundComboBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>scanTab</receiver>
   <slot>slot_dirtreeSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>basePathButton</sender>
   <signal>clicked()</signal>
   <receiver>scanTab</receiver>
   <slot>slot_basePathButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>homePathButton</sender>
   <signal>clicked()</signal>
   <receiver>scanTab</receiver>
   <slot>slot_homePathButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_scanButtonClicked()</slot>
  <slot>slot_homePathButtonClicked()</slot>
  <slot>slot_basePathButtonClicked()</slot>
  <slot>slot_virusFoundComboBoxChanged()</slot>
  <slot>slot_recursivScanCheckBoxClicked()</slot>
  <slot>slot_abortScan()</slot>
  <slot>slot_dirtreeSelectionChanged()</slot>
 </slots>
</ui>
